integration_tests/tests/consistency/row_comparisons/consistency_event.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+-- this test ensures the daily_activity end model matches the prior version
+with prod as (
+    select *
+    from {{ target.schema }}_mixpanel_prod.mixpanel__event
+),
+
+dev as (
+    select *
+    from {{ target.schema }}_mixpanel_dev.mixpanel__event
+),
```

Comment:
The `properties` and `event_properties` fields have the potential to be interpreted as JSON/SUPER in certain warehouses (as opposed to JSON formatted string). JSON datatypes are not supported in except distinct operations. Therefore, we should remove these fields from the comparison so we do not encounter false failures in the future.

![image](https://github.com/user-attachments/assets/34ff9256-5678-4456-96a6-2d4d908b6f90)


```suggestion
{% set exclude_fields = ["properties", 'event_properties'] %}
{% set fields = dbt_utils.star(from=ref('mixpanel__event'), except=exclude_fields) %}

-- this test ensures the daily_activity end model matches the prior version
with prod as (
    select {{ fields }}
    from {{ target.schema }}_mixpanel_prod.mixpanel__event
),

dev as (
    select {{ fields }}
    from {{ target.schema }}_mixpanel_dev.mixpanel__event
),
```

Reply:
Thank you--updated!